.github/workflows/ci.yml

```diff
@@ -19,8 +19,20 @@ jobs:
       # Required to create loop devices.
       options: --privileged
     steps:
-    - uses: actions/checkout@v2
-    - name: Run storage tests
+    - uses: actions/checkout@v3
+    - name: Run storage tests not marked as root as vdsm user
+      run: ./automation/tests-storage.sh vdsm -u=vdsm
+  test-storage-root:
+    env:
+      TRAVIS_CI: 1
```

Comment:
are the errors/skips still relevant? we don't run it in travis any more

Reply:
Looks like legacy env, true. I will remove them for the updated parts in this patch, so that it is not carried over. But if removing it from the CIs is the right action to take, I would do it in a separate patch.

Reply:
https://github.com/oVirt/vdsm/issues/279